api/end_device.proto

```diff
@@ -199,13 +203,13 @@ message MACSettings {
   RxDelayValue rx1_delay = 6;
   // Rx1 data rate offset.
   // If unset, the default value from Network Server configuration will be used.
-  google.protobuf.UInt32Value rx1_data_rate_offset = 7 [(validate.rules).uint32.lte = 7];
+  DataRateOffsetValue rx1_data_rate_offset = 7;
```

Comment:
Doesn't this break API compatibility? A `google.protobuf.UInt32Value` renders to JSON as a number. A `DataRateOffsetValue` seems to render to JSON as an object.